src/provider/persistence/dataLayer/IndexedDb.ts

```diff
@@ -168,19 +168,19 @@ export class IndexedDb<T extends Entity> implements IDatastore<T> {
      * @param {string} tableName The name of the table to perform
      * @param {string} uuid The uuid of the app
      * @public
-     * @returns {Promise<boolean>} A value of whether it was successfully created or not
+     * @returns {Promise<number>} A value of whether it was successfully created or not
```

Comment:
Same change as IDatastore - "A value of whether it was successfully created or not" -> "The count of removed entries"